src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/CustodyKeyDatesAPITest.java

```diff
@@ -73,6 +75,13 @@ public void setup() {
         jdbcTemplate.execute("DELETE FROM CONTACT");
     }
 
+    @After
```

Comment:
clean up after dirtying the database. This will be replaced by looking at test transactions